### PR TITLE
Add documentation about History and the Staging tab

### PR DESCRIPTION
**Pull request by saraycp**
It also re-orders the sections.

### CODE REVIEW

**Comment by saraycp** [comment]
[Documentation](https://trello.com/c/KZwAiKTy/1069-documentation)

File: xml/obs_best_practice_staging_workflow.xml [hunk 1]
```diff
@@ -8,16 +8,22 @@
  xmlns:xi="http://www.w3.org/2001/XInclude"
  xmlns:xlink="http://www.w3.org/1999/xlink" >
   <title>Staging</title>
-  <para></para>
+  <para>
+    At the moment, it is possible to create a Staging for any kind of project
+    unless the project is already one of the staging projects.
+  </para>
+  <para>
+    All the Staging workflow starts in the tab called 'Staging' that we can
```
Comment (dmarcoux) [inline xml/obs_best_practice_staging_workflow.xml]: start<strike>s</strike> in the tab 'Staging' which can be found on the project's page.

File: xml/obs_best_practice_staging_workflow.xml [hunk 2]
```diff
@@ -154,6 +160,45 @@
...
     </note>
+    <sect3>
+      <title>Create Staging Project from Scratch</title>
+      <para>
+        Right after the creation of a Staging, two new staging projects are
+        automatically created and assigned to it: Staging:A and Staging:B. However,
+        it is also possible to create a new staging project from scratch.
+      </para>
+      <para>
+        Being placed on the Staging dashboard, click on
```
Comment (dmarcoux) [inline xml/obs_best_practice_staging_workflow.xml]: I would remove the "Being placed", it sounds weird.

**Comment by saraycp** [comment]
@dmarcoux I have addressed your feedback, thanks!

**Pull request merged**